src/components/common/styles/Sidebar.css

```diff
@@ -4,9 +4,12 @@
   justify-content: space-between;
 }
 
-.ant-switch-checked {
+/* This switch style, which does not appear to be in use, was overriding
+styling in the navbar. It is left here in case styling does need to use this
+class, in which case a fix can be made. */
+/* .ant-switch-checked {
   background-color: #BFBFBF !important;
-}
+} */
 
```

Comment:
I believe if you remove the `!important` flag on the background color parameters, that the override will stop.

Reply:
Perhaps, we did try different approaches. May simply remove this in the future.

Reply:
How can you check/confirm that this isn't being used elsewhere? @mcfrr3 has spent some good time resolving ant-design issues involving check-boxes + CSS bleeding from other components affecting his own work. I'll message him about this and ask him to look. Might be worth a discussion with him if you haven't had one yet. It would be better to have something more definitive than commenting it out: something like Melody's thought, or removal outright.

Reply:
Addressed this, insight from @mcfrr3 as well on future considerations